.eslintrc

```diff
@@ -0,0 +1,11 @@
+plugins:
+  - fxa
```

Comment:
We need to run `npm run lint` on travis as well 

https://github.com/mozilla/fxa-shared/blob/master/.travis.yml


Example: https://gist.github.com/ronapelbaum/ef3c3459f01b002d0d719ca205401174#file-travis-yml-L6